lib/rdoc/rubygems_hook.rb

```diff
@@ -310,5 +314,9 @@ def self.generation_hook installer, specs
       # Generate document for compatibility if this is a default gem.
       RubyGemsHook.generate(installer, specs)
     end
+
+    def self.load_rdoc
+      @rdoc_version = RubyGemsHook.load_rdoc
+    end
```

Comment:
How about delegating `load_rdoc` and `rdoc_version` entirely?

```suggestion

    def self.load_rdoc
      RubyGemsHook.load_rdoc
    end

    def self.rdoc_version
      RubyGemsHook.rdoc_version
    end
```